tests/python/mock_jira.py

```diff
@@ -1619,28 +1630,46 @@ def transitions(self, *args, **kwargs):
         """
         Mocked Jira method.
         """
-        return []
+        return [{
+            "id": 1,
+            "name": "From Fake",
+            "to": {
+                "name": "To Do"
+            },
+        }]
+    
+    def transition_issue(self, *args, **kwargs):
+        return ""
 
     def search_assignable_users_for_issues(
-        self, name, startAt=0, maxResults=20, *args, **kwargs
+        self, username=None, query=None, startAt=0, maxResults=2, *args, **kwargs
```

Comment:
Why do we change maxResults from 20 to 2?

Reply:
Good catch. Reverted.